src/test/kotlin/uk/gov/dluhc/notificationsapi/service/StatisticsUpdateServiceTest.kt

```diff
@@ -166,11 +172,48 @@ class StatisticsUpdateServiceTest {
         val applicationId = aRandomSourceReference()
 
         // When
-        statisticsUpdateService.triggerStatisticsUpdate(applicationId, SourceType.VOTER_CARD)
-        statisticsUpdateService.triggerStatisticsUpdate(applicationId, SourceType.POSTAL)
-        statisticsUpdateService.triggerStatisticsUpdate(applicationId, SourceType.PROXY)
+        statisticsUpdateService.triggerStatisticsUpdate(applicationId, SourceType.VOTER_CARD, false)
+        statisticsUpdateService.triggerStatisticsUpdate(applicationId, SourceType.POSTAL, false)
+        statisticsUpdateService.triggerStatisticsUpdate(applicationId, SourceType.PROXY, false)
+
+        // Then
+        verifyNoInteractions(triggerOverseasApplicationStatisticsUpdateQueue)
+    }
+
+    @Test
+    fun `should send a message to the applications queue for appplications flagged for OS`() {
```

Comment:
We could iterate over `SourceType` with a `@ParameterizedTest` so that if a new source type is added, this test will fail unless the service is updated